setup.py

```diff
@@ -376,17 +376,19 @@ def get_data_files():
     },
     'install_requires': [
         'async_generator',
-        'jupyter_server>=0.1.0,<0.2.0',
-        'nbconvert>=5.5.0,<6',
+        'jupyter_server>=0.3.0',
+        'jupyter_client>=6.1.3',
+        'nbclient>=0.2.0',
+        'nbconvert==6.0.0a1',
```

Comment:
Do we really need 6.0.0 or is this merely for testing?

Reply:
We need nbconvert `6.0.0.a1` for async jinja template execution.

Reply:
Thanks